.prettierignore

```diff
@@ -11,3 +11,4 @@ playground/html/invalid.html
 playground/html/valid.html
 playground/external/public/slash@3.0.0.js
 playground/ssr-html/public/slash@3.0.0.js
+playground/worker/classic-iife.js
```

Comment:
`prettier-ignore` comment doesn't seem to avoid prepending `;`  to `(() => {})()`, so I needed to prettierignore an entire file.

Technically we could test the same thing by just adding `(() => {})()` to existing `playground/worker/classic-worker.js`, but that would require prettierignore-ing that file, so I added a new file `playground/worker/classic-iife.js` just for this.